docs/sources/setup/installation.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 'Installation'
+menuTitle: Installation
+description: Install the GitHub data source plugin
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+weight: 101
+---
+
+# Install the GitHub data source plugin
+
+There are multiple ways to install this plugin into your Grafana instance
+
+## Install from plugin catalog
+
+1. Login to Grafana Web UI as **Server Admin**.
```

Comment:
Using the same language as in https://grafana.com/docs/grafana/latest/administration/user-management/server-user-management/#:~:text=of%20users%3A-,Sign%20in%20to%20Grafana%20as%20a%20server%20administrator.,-Click%20Administration%20in.

```suggestion
1. Sign in to Grafana as a server administrator.
```